pwndbg/aglib/remote.py

```diff
@@ -0,0 +1,54 @@
+"""
+Information about whether the debuggee is local (under GDB) or remote
```

Comment:
Shall we say, under GDB or LLDB?

Reply:
Yes. There are probably a few of these comments remaining in `aglib` that reference GDB specifically, but that mean both of them.